### ISSUE TITLE
Extract the component from the bundle

### ISSUE THREAD

**Issue by sagikazarmark** [akeneo/BatchBundle#88]
This library looks pretty cool. However it is closely tied to symfony. Is it possible to extract common components so that it can be used "on it's own'?

I am participating in a similar project: https://github.com/portphp/

Wonder if we could bring them close together.

**Comment by nidup** [comment]
Thank you for this interesting feedback!

At the very beginning, we used only Bundles in Akeneo PIM and we encountered some issues to re-use business code which was too much coupled with Symfony Framework and Doctrine ORM.

Then we started to introduce Components to write our very new business code (no coupled to Symfony Framework and Doctrine ORM) but to avoid large BC breaks we didn't extract all the business code from our existing bundles.

FYI, we're seriously thinking of extracting a library from this BatchBundle but we don't have yet ETA.

The library https://github.com/portphp/ looks pretty interesting, thanks for sharing!
